crawl-ref/source/mon-info.h

```diff
@@ -174,6 +174,7 @@ enum monster_info_flags
     MB_SLOWLY_DYING,
     MB_PINNED,
     MB_VILE_CLUTCH,
+    MB_HIGHLIGHT,
```

Comment:
Should reflect whatever name change you make to packed_cell.highlighted here as well.